content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Menjalankan Tugas-Tugas Otomatis dengan CronJob
+min-kubernetes-server-version: v1.8
+content_template: templates/task
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kamu dapat menggunakan {{< glossary_tooltip text="CronJob" term_id="cronjob" >}} untuk menjalankan {{< glossary_tooltip text="Job" term_id="job" >}} yang dijadwalkan berbasis waktu. Job akan berjalan seperti pekerjaan-pekerjaan [Cron](https://en.wikipedia.org/wiki/Cron) di Linux atau sistem UNIX.
+
+CronJob sangat berguna untuk membuat pekerjaan yang berjalan secara periodik dan berulang, misalnya menjalankan pencadangan atau mengirim email.
+CronJob  juga dapat menjalankan pekerjaan individu pada waktu tertentu, misalnya jika kamu ingin menjadwalkan sebuah pekerjaan dengan periode aktifitas yang rendah.
+
+CronJob  memiliki keterbatasan dan kekhasan.
+Misalnya, dalam keadaan tertentu, sebuah CronJob  dapat membuat banyak Job.
+Karena itu, Job haruslah _idempotent._
+
+Untuk informasi lanjut mengenai keterbatasan, lihat [CronJob](/docs/concepts/workloads/controllers/cron-Job).
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* {{< include "task-tutorial-prereqs.md" >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Membuat Sebuah CronJob
+
+CronJob  membutuhkan sebuah berkas konfigurasi.
+Ini adalah contoh dari berkas konfigurasi CronJob `.spec` yang akan mencetak waktu sekarang dan pesan hallo setiap menit:
+
+{{< codenew file="application/job/cronjob.yaml" >}}
+
+Jalankan contoh CronJob menggunakan perintah berikut:
+
+```shell
+kubectl create -f https://k8s.io/examples/application/job/cronjob.yaml
+```
+Keluaran akan mirip dengan ini:
+
+```
+cronjob.batch/hello created
+```
+
+Atau kamu dapat menggunakan `kubectl run` untuk membuat sebuah CronJob tanpa menulis sebuah konfigurasi yang lengkap:
+
+```shell
+kubectl run hello --schedule="*/1 * * * *" --restart=OnFailure --image=busybox -- /bin/sh -c "date; echo Hello from the Kubernetes cluster"
+```
+
+Setelah membuat sebuah CronJob, untuk mengecek statusnya kalian dapat menggunakan perintah berikut:
```

Comment:
kamu or kalian? let's keep consistent with the rest of page
```suggestion
Setelah membuat sebuah CronJob, untuk mengecek statusnya kamu dapat menggunakan perintah berikut:
```

Reply:
done